.github/workflows/deploy.yml

```diff
@@ -19,6 +19,13 @@ jobs:
         AWS_DEFAULT_REGION: us-east-1
         AWS_DEFAULT_OUTPUT: text
         GAMMA_CHIME_ENDPOINT: ${{secrets.GAMMA_CHIME_ENDPOINT}}
+        GAMMA_CHIME_ENDPOINT_IAD: ${{secrets.GAMMA_CHIME_ENDPOINT_IAD}}
```

Comment:
Have you set this yet?

Reply:
We would need a better name and lets not use the region short names here.